### PR TITLE
Actually get the team profile for the team

### PR DESCRIPTION
**Pull request by WardBenjamin**
Instead of sending a literal %s, send the formatted team_key string.

### CODE REVIEW

File: tbapy/main.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def team_profiles(self, team):
         :param team: Team to get data on.
         :return: List of Profile objects.
         """
-        return [Profile(raw) for raw in self._get('team/%s/social_media')]
+        return [Profile(raw) for raw in self._get('team/%s/social_media') % self.team_key(team)]
```
Comment (ErikBoesen) [inline tbapy/main.py]: This will not work either. Check your parentheses.

**Comment by ErikBoesen** [comment]
Thanks for pointing this out.

**Comment by WardBenjamin** [comment]
@ErikBoesen fixed, thanks

**Comment by ErikBoesen** [comment]
Rebase into one commit please.

**Pull request merged**